src/components/layout.scss

```
@@ -495,6 +499,11 @@ $dropdown-item-active-background-color: $primary;
           }
         }
 
+        .jump-anchor-header-padding {
+          // カード間の余白を均等にする
+          padding-top: calc(3.25rem + 1rem + 0.75rem);
```

これ何でずれるのか分かってなかったんですけどすごくよくわかりました、デフォルトのcolumnのpadding-topが消えちゃってるからなんですね！！

もしかしたら意図がわからなくなるかもなのでコメント書いとくのどうでしょう？
```suggestion
          padding-top: calc(3.25rem + 1rem + 0.75rem); // 元が3.25rem + 1rem、column分が0.75rem
```